tests/unit/moduleapi/blockonkeys.tcl

```diff
@@ -185,4 +185,32 @@ start_server {tags {"modules"}} {
         r fsl.push k 34
         assert_equal {34} [$rd read]
     }
+
+    test {Module client blocked on keys woken up by LPUSH} {
+        r del k
+        set rd [redis_deferring_client]
+        $rd blockonkeys.popall k
+        # wait until client is actually blocked
+        wait_for_condition 50 100 {
+            [s 0 blocked_clients] eq {1}
+        } else {
+            fail "Client is not blocked"
+        }
+        r lpush k 42
```

Comment:
i saw the command is named `popall`, don't you want to push several items with this `lpush`?

Reply:
Yes, good idea.